Applications/Viqui/viqui.ui

```diff
@@ -937,6 +938,11 @@
     <string>Generate Report...</string>
    </property>
   </action>
+  <action name="actionExportSvm">
+   <property name="text">
+    <string>Export SVM Model...</string>
```

Comment:
Missing accelerator. Also, per previous comment, I don't think we should use "SVM" in the UI. (Or internally, though that more or less follows from not using it in the UI.)

Reply:
It looks like 'm' is available... "`Export IQR &amp;Model`"? Also, I'm inclined to park this just under 'save query plan' (and before the separator) instead...